Week-2/Transition/index.html

```diff
@@ -0,0 +1,21 @@
+<html lang="en">
```

Comment:
@mr-rego Animation looks good. Good work. Going forward please create different branches for the assignments